src/TinkerServiceProvider.php

```diff
@@ -27,9 +27,11 @@ public function register()
 
         $this->commands(['command.tinker']);
 
-        $this->publishes([
-            __DIR__.'/../config/tinker.php' => $this->app['path.config'].DIRECTORY_SEPARATOR.'tinker.php',
-        ]);
+        if ($this->app->runningInConsole()) {
```

Comment:
I'm fine with that. I just wasn't sure if everyone would want the functionality.